### DISCUSSION TITLE
is it possibled for folders to rememeber sorting method?

### DISCUSSION THREAD

**Discussion by yorunoken** [sxyazi/yazi#1248] [1. Q&A] [answered]
for example I would set the sorting to created first in `~/Downloads`, but still have it on alphabetical in ~

**Comment by sxyazi** [comment, marked as answer]
You can achieve it through DDS atm, see https://github.com/sxyazi/yazi/issues/623#issuecomment-2096270843

**Reply by yorunoken** [reply]
Thank you.
